core/SymbolRef.h

```diff
@@ -535,7 +536,9 @@ class SymbolRef final {
     core::SymbolRef owner(const GlobalState &gs) const;
     core::Loc loc(const GlobalState &gs) const;
     bool isPrintable(const GlobalState &gs) const;
-    const InlinedVector<Loc, 2> &locs(const GlobalState &gs) const;
+    using LOC_store = InlinedVector<Loc, 2>;
+    const LOC_store &locs(const GlobalState &gs) const;
```

Comment:
Nonblocking, but this seems like a) a peculiar place to put this typedef (why do `SymbolRef`s care about the underlying storage of the locs?) and b) a weird way to constrain the interface of `locs`.  Returning an `absl::Span<Loc>` here would really be more appropriate, and that would enable dropping the typedef entirely or moving it to someplace more appropriate (`core/Symbols.h`, perhaps).

I do realize that the `locs` interface was pre-existing, so not that big a deal, but it does feel weird.